lib/core/userinfo/components/user-info.component.html

```diff
@@ -148,6 +148,7 @@
                                       class="adf-userinfo__detail-title">{{identityUser | fullName}}</span>
                                 <span class="adf-userinfo__detail-profile"
                                       id="identity-email"> {{identityUser.email}} </span>
+                                <a class="adf-userinfo__detail-profile" href="#/profile">My Profile</a>
```

Comment:
all the text must be coming from the i18n resources

Reply:
Done